src/Client.js

```diff
@@ -95,72 +95,12 @@ Client.prototype.paginate = function(expression, params) {
 };
 
 /**
- * Issues a HTTP `GET` request via the legacy REST API.
```

Comment:
Removing this means that we should release a major version and also document it as part of the CHANGELOG.